platform/linux-generic/pool/subsystem.c

```diff
@@ -0,0 +1,17 @@
+/* Copyright (c) 2017, ARM Limited. All rights reserved.
+ *
+ * Copyright (c) 2017, Linaro Limited
+ * All rights reserved.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+#include <odp_module.h>
+
+#define SUBSYSTEM_VERSION 0x00010000UL
```

Comment:
Yes, current only ODP_SUBSYSTEM_DEFINE() specifies this version info, the usage can come late with a design (maybe its too earlier to have it here).